tests/python/pytorch/graphbolt/test_minibatch.py

```diff
@@ -336,6 +336,76 @@ def test_get_dgl_blocks_hetero():
     assert result == expect_result
 
 
+def test_get_dgl_blocks_hetero_empty_edges():
+    csc_formats = [
+        {
+            relation: gb.CSCFormatBase(
+                indptr=torch.tensor([0, 1, 2, 3]),
+                indices=torch.tensor([0, 1, 1]),
+            ),
+            reverse_relation: gb.CSCFormatBase(
+                indptr=torch.tensor([0, 0, 0, 1, 2]),
+                indices=torch.tensor([1, 0]),
+            ),
+        },
+        {
+            relation: gb.CSCFormatBase(
+                indptr=torch.tensor([0, 1, 2]), indices=torch.tensor([1, 0])
+            ),
+            reverse_relation: gb.CSCFormatBase(
```

Comment:
so `FusedCSCSamplingGraph.sample_neighbors()` always returns sampled_csc for each edge type even it's empty? Could we mention this in docstring or documentation? probably in `sample_neighbors()`?

Reply:
Yes. Let me add docstring.